src/app.ts

```diff
@@ -17,8 +20,19 @@ validateEnv();
 const app = express();
 // const prisma = new PrismaClient();
 
+i18next
+  .use(Backend)
+  .use(middleware.LanguageDetector)
+  .init({
+    fallbackLng: "en",
+    backend: {
+      loadPath: "./locales/{{lng}}/translation.json",
+    },
+  });
+
```

Comment:
@aimee-annabelle  isn't there a better folder or files to configure this middleware outside of the entry point of the app. Like a middleware folder. Please check the project structure. otherwise you did a Good job

Reply:
I could try to find a way but if you have one you can help me because I didn't see another way to apply this in another file other than the entry point